packages/client/src/query/aggregations/AggregateOpts.ts

```diff
@@ -16,16 +16,17 @@
 
 import type { ObjectOrInterfaceDefinition } from "@osdk/api";
 import type {
+  AggregatableKeys,
   AggregationClause,
   GroupByClause,
   WhereClause,
 } from "../../query/index.js";
 
 export type AggregateOpts<
-  Q extends ObjectOrInterfaceDefinition<any, any>,
-  AC extends AggregationClause<Q>,
+  Q extends ObjectOrInterfaceDefinition,
+  KK extends AggregatableKeys<Q> = AggregatableKeys<Q>,
```

Comment:
I don't really think I need this with the way I'm using it right now, I can probably just have AggregationClause use `AggregatableKeys<Q> `directly in its type definition, since I'm just piping that through anyways